src/hooks/useAppBootstrap.functions.ts

```diff
@@ -0,0 +1,75 @@
+import Minilog from '@cozy/minilog'
+
+import { getErrorMessage } from '/libs/functions/getErrorMessage'
+
+const log = Minilog('useAppBootstrap.functions')
+
+interface OnboardingParams {
+  onboardUrl: string | null
+  fqdn: string | null
+}
+
+export const parseOnboardingURL = (
```

Comment:
I think as a pure function it should be unit tested

Reply:
I didn't add tests because those are already covered by `useAppBootstrap.spec.js`. I see this extraction more like an implementation detail. But this is pretty subjective. If I add unit tests, those will be kind of duplicates. 
What do you think about this?

Reply:
I agree with you, it can be considered an implementation detail and thus shouldn't be tested.